tests/evm/test_callop.py

```diff
@@ -146,30 +149,35 @@ def gen_testing_data():
 
 
 @pytest.mark.parametrize(
-    "opcode, caller, callee, caller_ctx, stack, is_warm_access, expected", TESTING_DATA
+    "opcode, caller, callee, parent_caller, parent_value, caller_ctx, stack, is_warm_access, expected",
+    TESTING_DATA,
 )
-def test_call_staticcall(
+def test_callop(
     opcode: Opcode,
     caller: Account,
     callee: Account,
+    parent_caller: Account,
+    parent_value: int,
     caller_ctx: CallContext,
     stack: Stack,
     is_warm_access: bool,
     expected: Expected,
 ):
-    is_call = opcode == Opcode.CALL
-    is_static = not is_call or stack.value == 0
-
     randomness = rand_fq()
 
+    is_call = opcode == Opcode.CALL
+    is_delegatecall = opcode == Opcode.DELEGATECALL
+
     value = stack.value if is_call else 0
+    is_static = value == 0
```

Comment:
@adria0 commented to fix as:
```
-    is_static = value == 0
+    is_static = opcode == Opcode.STATICCALL
```

Reply:
It seems that `is_static = (opcode == CALL && value == 0) || opcode == DELEGATECALL || opcode == STATICCALL` as [this comment](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/281#discussion_r984224240) in `STATICCALL` PR.

For this code, I fixed to:
```
value = stack.value if is_call else 0
is_static = value == 0
```

Reply:
Add a comment for this code as:
```
# Set `is_static == 1` for both DELEGATECALL and STATICCALL opcodes, and
# also when `stack.value == 0` for opcode CALL.
value = stack.value if is_call else 0
is_static = value == 0
```

May reference this discussion - https://github.com/privacy-scaling-explorations/zkevm-specs/pull/281#discussion_r985078691

It seems that `is_static == 1` if `value == 0` (value is always zero for both `DELEGATECALL` and `STATICCALL` opcodes).